packages/web-components/src/global/main.css

```diff
@@ -6,3 +6,11 @@ va-process-list li :is(h2, h3, h4, h5) {
   clear: none;
   padding-top: 0.3rem;
 }
+
+va-table > va-table-row[slot] button {
+  display: inline-flex;
+  align-items: center;
+}
+va-table > va-table-row[slot] button svg {
+  width: 2.4rem;
+}
```

Comment:
Could we put this in a va-table-slot.css file and import it the way we do with the breadcrumbs slot CSS above?

Reply:
Yeah, that's on my todo list.